src/main/ngapp/angular.json

```diff
@@ -13,7 +13,8 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "outputPath": "../webapp/ng",
+            "outputPath": "../webapp/",
+            "deleteOutputPath": false,
```

Comment:
what does deleting output path do?

Reply:
It just ensures we don't delete the contents of the folder the app is built in. This way, we don't delete the xml file in the webapp directory